api/workAllocation2/interfaces/caseRole.ts

```diff
@@ -12,3 +12,10 @@ export interface Action {
   id: string;
   title: string;
 }
+
+export enum RoleType {
+  LEAD_JUDGE = 'Lead Judge',
```

Comment:
there has been a similar TypeOfRole in src\role-access\models\allocate-role.enum